jetty-ee9/jetty-ee9-annotations/src/main/java/org/eclipse/jetty/ee9/annotations/AnnotationIntrospector.java

```diff
@@ -163,7 +164,7 @@ public boolean isIntrospectable(Object o, Object metaInfo)
             {
                 //must be from a descriptor. Only introspect if the descriptor with which it was associated
                 //is not metadata-complete
-                if (_context.getMetaData().isMetaDataComplete())
+                if (_context.getMetaData().isMetaDataComplete() && _context.getAttribute(ALLOW_INCOMPLETE_METADATA) == null)
```

Comment:
Needs a comment to describe what is happening here, especially because it is not spec compliant. I also don't think the name of the attribute really describes the situation either.  Also, I thought we were just going to apply this to ee8 only? I don't really want to pollute ee9 with this, particularly because it never had this behaviour in the first place.

Reply:
@olamy can't we selectively use some src files in `ee8`?

Reply:
The current implementation of the tooling is `ee8` maven module to contain only generated sources.
Not sure to understand what would you need? some local file in src/main/java to override the file coming from the translation.
Per my understanding ee9 is just ee8 with package renaming jakarta.servlet to javax.servlet and that's all
with exactly all the same features

Reply:
The default of this flag is always spec complaint.

So if it's not currently possible I don't think it's worth implementing for this.